yjit/src/core.rs

```diff
@@ -3229,6 +3234,8 @@ pub fn gen_entry_point(iseq: IseqPtr, ec: EcPtr, jit_exception: bool) -> Option<
     // Count the number of entry points we compile
     incr_counter!(compiled_iseq_entry);
 
+    CompilationLog::add_iseq(blockid);
```

Comment:
This place only hooks interpreter -> JIT calls, so it misses JIT -> JIT calls.

Since YJIT is not a method JIT and a compilation unit is a basic block, I think it should log blocks instead of methods. Otherwise, it would hide some long-tail compilations we probably don't want to miss.

Reply:
That was my misunderstanding. I thought we wanted to keep the volume down. I'm happy to add it.

Reply:
I've removed logging of method entry points and only log in `gen_single_block`, mostly duplicating the log message format already in use by `dump_disasm` feature. We can tweak this as needed.